packages/@aws-cdk/aws-cloudformation-codepipeline/test/integ.pipeline.ts

```diff
@@ -0,0 +1,42 @@
+import codepipeline = require('@aws-cdk/aws-codepipeline');
+import { ArtifactPath } from '@aws-cdk/aws-codepipeline';
+import { Role } from '@aws-cdk/aws-iam';
+import s3 = require('@aws-cdk/aws-s3');
+import cdk = require('@aws-cdk/cdk');
+import { ServicePrincipal } from '@aws-cdk/cdk';
+
+import cfn_codepipeline = require('../lib');
+
+const app = new cdk.App(process.argv);
+
+const stack = new cdk.Stack(app, 'aws-cdk-codepipeline-cloudformation');
+
+const pipeline = new codepipeline.Pipeline(stack, 'Pipeline');
+
+const sourceStage = new codepipeline.Stage(pipeline, 'Source');
+const bucket = new s3.Bucket(stack, 'PipelineBucket', {
+    versioned: true,
+});
+const source = new codepipeline.AmazonS3Source(sourceStage, 'Source', {
+    artifactName: 'SourceArtifact',
+    bucket,
+    bucketKey: 'key',
+});
+
+const cfnStage = new codepipeline.Stage(pipeline, 'CFN');
+
+const changeSetName = "ChangeSetIntegTest";
+const stackName = "IntegTest-TestActionStack";
+
+const role = new Role(stack, 'CfnChangeSetRole', {
```

Comment:
Looks like the default role

Reply:
It is, now we're integ-testing both variants :).